pom.xml

```diff
@@ -5,14 +5,14 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>ru.itis</groupId>
-    <artifactId>wish_list</artifactId>
+    <artifactId>wish-list</artifactId>
```

Comment:
Why renaming the project?

Reply:
best practices of maven module naming

Reply:
I don't think this is a module, any proof?